src/briefcase/commands/create.py

```diff
@@ -315,6 +316,21 @@ def install_app_support_package(self, app: BaseConfig):
                 self.logger.info(f"Using support package {support_package_url}")
 
             if support_package_url.startswith(("https://", "http://")):
+                if custom_support_package:
+                    # If the support package is custom, cache it using a hash of
+                    # the download URL. This is needed to differentiate to support
+                    # packages with the same filename, served at different URLs.
```

Comment:
If a `revision` argument is added to the URL below, should it be included in the hash? The standard BeeWare URL always returns different filenames for different revisions., but a custom URL might not.

Reply:
Good catch. Fixed.